notebooks/spark/AerospikeSparkPython.ipynb

```diff
@@ -5,7 +5,7 @@
    "metadata": {},
    "source": [
     "# Aerospike Connect for Spark Tutorial for Python\n",
-    "## Tested with Spark connector 3.0.1, Java 8, Apache Spark 3.0.0, Python 3.7  and Scala 2.12.11 and Spylon ( https://pypi.org/project/spylon-kernel/)"
+    "## Tested with Spark connector 2.8.0, Java 8, Apache Spark 2.4.0, Python 3.7  and Scala 2.11.12 and Spylon ( https://pypi.org/project/spylon-kernel/)"
    ]
   },
```

Comment:
Lets remove this whole section: "Here we download the Aerospike Spark jar"

What if the user wants to run a particular version of the connector that is not the latest? We need to provide that option.

Please provide the instructions on how to download it manually.